content/updates/2021-05/4-Product-completeness-in-the-API.md

```diff
@@ -0,0 +1,37 @@
+---
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Highlight the most enriched products on your channels
+::: meta-data type="Improvement" features="Productivity" available="late May" in="EE,GE" link-to-doc="https://api.akeneo.com/concepts/products.html#product"
+
+When developing your connectors, you can now ask for the product completeness whenever retrieving products. Thanks to this piece of information, you can easily push forward the most enriched products on your different channels.
+
+Also, if your PIM is connected to your BI tool, the product completeness can be very useful to follow KPIs such as ...
```

Comment:
@meganelepalud-akeneo We need examples of KPIs that could be shown in BI tools of our customers. Thanks!

Reply:
the product completeness percentage per family or the number of products under a specific percentage of completeness.

Reply:
```suggestion
Also, if your PIM is connected to your BI tool, the product completeness can be very useful to follow KPIs such as the product completeness percentage per family for example.
```